lib/integration/nation_builder.rb

```diff
@@ -2,7 +2,7 @@ class Integration::NationBuilder
 
   STANDARD_HEADERS = {'Accept' => 'application/json', 'Content-Type' => 'application/json'}
   ENDPOINTS = {
-    lists:           '/api/v1/lists/?per_page=100', #100 is max
+    list_count_page: '/supporter_counts_for_website',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.